test/AllReduceTests.cpp

```diff
@@ -193,7 +193,8 @@ namespace RcclUnitTesting
     for (int isMultiProcess : testBed.ev.GetIsMultiProcessList())
     {
       int const numProcesses = isMultiProcess ? totalRanks : 1;
-      testBed.InitComms(TestBed::GetDeviceIdsList(numProcesses, totalRanks));
+      const std::vector<int>& gpuPriorityOrder = testBed.ev.GetGpuPriorityOrder();
+      testBed.InitComms(testBed.GetDeviceIdsList(numProcesses, totalRanks, gpuPriorityOrder));
```

Comment:
These no longer have to be called on the `testBed` instance. They are static, so can be called with `TestBed::`. This is technically fine but is not idiomatic C++.

Reply:
Updated the code with your suggestion.